sfepy/terms/terms_elastic.py

```diff
@@ -636,3 +636,77 @@ def get_eval_shape(self, ts, mat0, mat1, parameter,
         out = CauchyStressTerm.get_eval_shape(self, mat0, parameter, mode,
                                               term_mode, diff_var, **kwargs)
         return out
+
+class NonsymetricElasticTerm(Term):
+    r"""
+    Elasticity term with non-symetric gradient. The indices of matrix
+    :math:`D_{ijkl}` are ordered as
+    :math:`[11, 12, 13, 21, 22, 23, 31, 32, 33]` in 3D and as
+    :math:`[11, 12, 21, 22]` in 2D.
+
+    :Definition:
+
+    .. math::
+        \int_{\Omega} \uul{D} \ul{u} : \ul{v}
+
```

Comment:
There are several typos here: symetric -> symmetric, \uul{D} -> \ull{D} (try building html docs...)
Is the math definition correct? I would assume gradients of u and  v there.

Concerning the class name, what about just LinearElasticNSGTerm or NonsymElasticTerm?

The ordering of components might be better to be a super-set of the symmetric ordering (11, 22, 33, 12, 13, 23, 21, 31, 32), so that e.g. the trace is easier to extract, but as you wish.